code-review.md

```diff
@@ -0,0 +1,43 @@
+# Implementing a Strong Code Review Culture
+
+## Abstract
+
+Code review is about catching bugs, right? Wrong. Modern code review is about
+socialization, learning, and teaching.
+
+What's the true value of code review? How can I get the most out of a peer's
+review and how can I review code without being seen as overly critical? Code
+review is a skill that will make you a better developer and a better teammate.
```

Comment:
Code review is a skill. Seems like this should be:

Reviewing code is a skill
Performing code reviews well is a skill